### PR TITLE
Adds: Prerequisites page for Tutorial

### PR DESCRIPTION
**Pull request by amanjeev**
This PR adds a new page - Important Prerequisites to the Tutorials section for the Community Edition to begin with.

This is needed because many of those details are scattered and some are easy to miss. This page adds those in one page before anyone embarks on the journey to create a Community Edition Tyk setup.

### CODE REVIEW

File: tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+},
+...
+```
+
+## Files vs API
+
+For Community Edition, note that you will need to know when to use the Tyk API
+endpoint and when mere placing files will do. So far
+
+- Files: Policies, Web APIs
```
Comment (letzya) [inline tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark]: We usually use the term “api definition”

File: tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+weight: 1
+---
+These are some common settings that you need before proceeding with other parts of this tutorial.
+
+{{% tabs %}}
+{{% tab "Community Edition" %}}
+
+## Tyk Config
+
+### Path to Web API configurations directory
```
Comment (letzya) [inline tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark]: We use the term api definition

File: tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+...
+```
+
+## Files vs API
+
+For Community Edition, note that you will need to know when to use the Tyk API
+endpoint and when mere placing files will do. So far
+
+- Files: Policies, Web APIs
+- Tyk API: API key generation
```
Comment (letzya) [inline tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark:60]: And api definition and reload

File: tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark]: Done

File: tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/try-out-tyk/tutorials/important-prerequisites.mmark]: Done

**Pull request merged**